src/hiccup/form.clj

```diff
@@ -74,6 +74,13 @@
              :value value
              :checked checked?}]))
 
+(defn- selected? [x selected]
+  (boolean
+    (cond
+      (sequential? selected) ((set selected) x)
+      (ifn? selected) (selected x)
+      :else (= x selected))))
```

Comment:
Almost perfect! Could you change this to:

```clojure
  (boolean (cond
             (sequential? selected) ((set selected) x)
             (ifn? selected) (selected x)
             :else (= x selected))))
```

Some Clojure editors default to using two spaces to indent functions, but Hiccup uses the style in the Clojure Style Guide, so:

```clojure
(foo
 (bar x)
 (baz y))
```

Or:

```clojure
(foo (bar x)
     (baz y))
```

Two-space indentation is reserved for control-flow macros like `cond`. I should probably add a formatting step to the GitHub workflow to make this automatic.

Reply:
Ah, I see.  Done.